IFTPipeline.jl/src/feature-extraction.jl

```diff
@@ -90,3 +93,93 @@ function extractfeatures(;
     serialize(joinpath(output, "floe_props.jls"), props)
     return nothing
 end
+
+function extractfeatures_single(;
+    input::String, output::String, minarea::Int64, maxarea::Int64, features::Array{String}
+)
+    @info "Loading segmented floes from $input"
+    labeled_floes = Int.(load_labeled_img(input))
+
+    @info "Extracting features from each floe: $features"
+    props = IFTPipeline.extractfeatures(labeled_floes; minarea=minarea, maxarea=maxarea, features=features)
+
+    @info "Extracted properties:"
+    @info props
+
+    @info "Writing to $output"
+    FileIO.save(output, props)
+    return nothing
+end
+
+"""
+    load_labeled_img(path)
+
+Load an unsigned integer image from a file.
+
+See also: save_labeled_img
+"""
+function load_labeled_img(path::AbstractString)
+    image = FileIO.load(path)
+    image_reinterpreted = convert_uint_from_gray(image)
+    return image_reinterpreted
+end
+
+"""
+    save_labeled_img(image, path)
+
+Save an unsigned integer image to an image file.
+
+See also: load_labeled_img
+"""
+function save_labeled_img(image::AbstractArray{<:Integer}, path::AbstractString)
+    image_reinterpreted = convert_gray_from_uint(image)
+    FileIO.save(path, image_reinterpreted)
+    return path
+end
+
+int_to_fixedpoint_map = Dict(
```

Comment:
Use all capital for constants (more julian).

Reply:
Done